frontend/src/components/pages/RegistrantExperience/Waiver/index.tsx

```diff
@@ -1,13 +1,136 @@
+        <Stack spacing={5} direction="row">
+          <Radio value="true" mb="0">
+            {/* NOTE!?: I use mb='0' because there's a margin misalignment issue with Radio component */}
+            I agree
+          </Radio>
+          <Radio value="false">
+            I{" "}
+            <Text as="span" fontWeight="bold">
+              do not
+            </Text>{" "}
+            agree
+          </Radio>
+        </Stack>
+      </RadioGroup>
+    </VStack>
+  ));
+
+  const requiredClausesJSX: ReactJSXElement[] = Array.from(
+    waiverInterface.requiredClauses,
+  ).map((clause: RequiredClauseResponse, index: number) => (
+    <ListItem key={index}>{clause.text}</ListItem>
+  ));
 
-const Waiver = (): React.ReactElement => {
   return (
     <Box>
-      <Text textStyle="displayXLarge">Camp Registration</Text>
-      <Text>Waiver</Text>
+      <VStack spacing={8} pt="1" align="stretch">
+        <Text textStyle="displayXLarge">Camp Registration</Text>
+        <Text textStyle="displayLarge">{waiverInterface.campName}</Text>
+        <Text textStyle="displayLarge" color="primary.green.100">
+          Waiver
+        </Text>
+        <Text textStyle="displayMedium" fontWeight="bold">
+          In consideration of the participation of my child/children, (the
+          “child or children“), in the Focus on Nature photography camp/workshop
+          and all activities associated therewith,
+        </Text>
+        <Text fontWeight="bold">
+          I, the undersigned parent/guardian of the child/children, agree to the
+          following terms and conditions:
+        </Text>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text color="primary.green.100" fontWeight="bold">
+          Identity and Waiver of Liability
+        </Text>
+
+        <OrderedList pl="10">{requiredClausesJSX}</OrderedList>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text fontWeight="bold">
+          By clicking the checkbox below, you agree to the terms and conditions
+          above.
+        </Text>
+        <Checkbox
+          isChecked={waiverInterface.agreedRequiredClauses}
+          onChange={() =>
+            waiverDispatch({ type: WaiverActions.ClICK_REQUIRED_CLAUSE })
+          }
+        >
+          I agree to the above identity and liability waiver conditions{" "}
+          <Text as="span" color="red">
+            *
+          </Text>
+        </Checkbox>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text color="primary.green.100" fontWeight="bold">
+          Additional Clauses
+        </Text>
+        {optionalClausesJSX}
+      </VStack>
+
+      <FormControl pt="9" width="40%">
+        <FormLabel fontWeight="bold">Name</FormLabel>
+        <Input fontFamily="cursive" />
+      </FormControl>
+
+      <FormControl pt="5" width="40%">
+        <FormLabel fontWeight="bold">Date</FormLabel>
+        <Input />
+      </FormControl>
```

Comment:
Think the formatting is off here too?

<img width="1423" alt="Screen Shot 2022-11-26 at 3 51 22 AM" src="https://user-images.githubusercontent.com/40974372/204080645-1071f47c-96cf-4429-a612-ce54c32346da.png">

Reply:
Yes, I'll change it to the dev handoff.